### DISCUSSION TITLE
Enable subcategories & multi-tag 子分类,多标签

### DISCUSSION THREAD

**Discussion by D-K-Deng** [saicaca/hexo-theme-vivia#26] [Q&A] [answered]
想问一下子分类是如何实现的，比如示例网站下的Bar 和 Baz分类
<img width="236" alt="1695570106768" src="https://github.com/saicaca/hexo-theme-vivia/assets/126566259/60db643e-3fd0-48a1-9a5a-5f9aec0f97dc">

**Comment by D-K-Deng** [comment]
还有同一篇多个tag是如何实现的呢？

**Comment by saicaca** [comment, marked as answer]
参考 Hexo 官方文档 https://hexo.io/zh-cn/docs/front-matter.html

```yaml
categories:
- Foo
- Bar
- Baz
```
或者
```yaml
categories: [Foo, Bar, Baz]
```
两者皆可；添加多个 Tag 同理。
